packages/components/src/Combobox/components/ComboboxContent/ComboboxList.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { SetStateAction } from "react";
```

Comment:
There are folders for ComboboxAction, ComboboxContent, ComboboxTrigger. There are sub-folders in ComboboxTrigger for ComboboxTriggerButton and ComboboxTriggerChip. Big fan of the organization.

My naive new person question though: in ComboboxContent, we have ComboboxList and ComboboxSearch. Should those actually be in their own top-level folders? Or should they be in sub-folders under ComboboxContent but prefixed like ComboboxTrigger? Or is there another pattern at play that I'm completely missing (that's entirely possible and most likely).

Reply:
it's like you read my mind!

https://github.com/GetJobber/atlantis/pull/1550#issuecomment-1739910249

still looking for feedback on this topic